server/service/src/sync/test/test_data/stock_line.rs

```diff
@@ -156,8 +156,8 @@ fn item_line_2_pull_record() -> TestSyncPullRecord {
             pack_size: 1,
             cost_price_per_pack: 0.0,
             sell_price_per_pack: 0.0,
-            available_number_of_packs: -1000,
-            total_number_of_packs: -1001,
+            available_number_of_packs: -1000.0,
```

Comment:
hmm, this is not part of you change, but available and total would never be negative (or should never be) for stock_line, could you please change.

Reply:
all changes except above test [e59545b](https://github.com/openmsupply/open-msupply/pull/642/commits/e59545b967d039c2128041d4172062106a0dc820)